### ISSUE TITLE
Not working with Apollo fork of Sunshine

### ISSUE THREAD

**Issue by junglejuz** [OneLiberty/moonlight-chrome-tizen#83]
- Tizen Version 1651 (Samsung S95BA)
- Docker installation

Using latest Apollo version of sunshine on host PC. (https://github.com/ClassicOldSong/Apollo)

Moonlight Windows client can pair and connect no problem..
Tizen client does not with pair with Apollo. It does pair with vanilla Sunshihne though.

Message on the TV after entering pairing code says "Error: failed to pair with [host]"

message in Apollo says "Pair success! Please grant necessary permissions to the client manually below", but the device is not available in the list.

**Comment by erlingpaulsen** [comment]
I'm also using Apollo instead of vanilla Sunshine. Very interested to get it to work with the Tizen client as well.

**Comment by Seekuhnetz** [comment]
@erlingpaulsen @junglejuz
This is really interesting because i am using Apollo and this Fork of Monnlight for Tizen since November on my Samsung S90C with no problems.
So either something has changed since then or it is different per TV / OS Version?
I am still on Tizen 7 and Version 2000.
I just got an upgrade to Tizen 8 and Version 6000 or something and now i am afraid to upgrade to not destroy something.
By the way i can remember that i had to remove the Sunshine Server in Moonlight and restart Moonlight afterwards.
Otherwise it would not let me connect.
I think it got confused because the IP of my host did not change of course.
So maybe try removing your old Sunshine Server first before adding the Apollo one?
If this does not help i can provide the exact version numbers if you want.

**Comment by Sebasss97** [comment]
I'm facing the same issue on my crystal au9070 tv. I'd really like to use apollo instead of sunshine for the integrated Vdd

**Comment by OneLiberty** [comment]
As stated in apollo's github, 

> [Apollo](https://github.com/ClassicOldSong/Apollo) and [Artemis(Moonlight Noir)](https://github.com/ClassicOldSong/moonlight-android) will no longer be compatible with OG Sunshine and OG Moonlight eventually, but they'll work even better with much more carefully designed features.

This might already be the case reading you all. 

I'm not planning on supporting it for now, nor in the near future. 
This client is intended to use with either GS (not recommended) or sunshine. 

Closing this as not planned for now
Feel free to reopen it if needed

**Comment by ClassicOldSong** [comment]
TBH pairing process for Apollo doesn't differ from Sunshine at all. It's far from being incompatible for Apollo in the current phase, so might you try on your TV and share the log from your client?

**Comment by ClassicOldSong** [comment]
If it does not appear in the client list that means the pair isn't acutally successful. Maybe some steps are out of order.

**Comment by erlingpaulsen** [comment]
I have tried myself now, and it works for me. The Frame 2024 model, so Tizen 8.

**Comment by ClassicOldSong** [comment]
I think I know where the problem lies. The key generated by the SSL library on older Tizen OS cannot pass newer SSL verifications. So if you can upgrade, make sure to use the latest firmware.

**Comment by Melvincraft** [comment]
> I think I know where the problem lies. The key generated by the SSL library on older Tizen OS cannot pass newer SSL verifications. So if you can upgrade, make sure to use the latest firmware.

I already have the most recent update for my S95B (ver 1651.7). Is there any way of making it compatible?

**Comment by ClassicOldSong** [comment]
I don't own Samsung TVs so idk when this version was released. But it seems that it is still too old and not containing correctly implemented SSL support.

**Comment by dos-ise** [comment]
@ClassicOldSong i don't think this is exclusive to Apollo. Had the same issue with the latest sunshine release. Therefore i downgraded and that works fine.

**Comment by ClassicOldSong** [comment]
We should @OneLiberty again as they closed the issue for thinking it's "Apollo's incompatibility with Moonlight"

**Comment by dos-ise** [comment]
I have created this issue.
https://github.com/OneLiberty/moonlight-chrome-tizen/issues/92
